src/components/address-book/EntryDialog/index.tsx

```diff
@@ -38,7 +39,16 @@ const EntryDialog = ({
     defaultValues,
     mode: 'onChange',
   })
-  const { handleSubmit, formState } = methods
+
+  const { handleSubmit, formState, watch, trigger } = methods
+
+  const address = watch('address')
+
+  useEffect(() => {
+    if (address) {
+      trigger('address')
+    }
+  }, [address, trigger])
```

Comment:
I would suggest doing the revalidation inside the `AddressInput` component with the `shouldValidate` option when setting the value.

```
const setAddressValue = useCallback(
    (value: string) => setValue(name, value, { shouldValidate: true }),
    [setValue, name],
  )
```